APITests/tests/index.json

```diff
@@ -18,6 +18,34 @@
                     }
                 ]
             },
+            {
+                "section": "§4.2.5. URLs",
+                "href": "https://www.w3.org/TR/pub-manifest/#value-url",
+                "tests": [
+                    {
+                        "id": "m4.2.5.01",
+                        "description": "In case of an embedded manifest, 'base' is the URL of the embedded document",
+                        "actions": "The absolute value of the chapter url should be relative to the URL of the embedding document",
+                        "errors": "none",
+                        "media-type": "text/html"
+                    },
+                    {
+                        "id": "m4.2.5.02",
+                        "description": "In case of an embedded manifest, 'base' is the value of the base tag of the embedded document",
+                        "actions": "The absolute value of the chapter url should be 'https://www.example.org/chapter1.html' in the the document",
+                        "errors": "none",
+                        "media-type": "text/html"
+                    },
+                    {
+                        "id": "m4.2.5.03",
+                        "description": "In case of an linked manifest, 'base' is the URL of the linked JSON manifest",
+                        "actions": "The absolute value of the chapter url should be relative to url of the linked manifest",
```

Comment:
While this is technically correct to the spec prose, I thought we ultimately decided it was the opposite that was true - that the base url was always the base of the referencing document (i.e., regardless of embedding/linking). Wasn't that partly why we restricted the PEP to being the only document that could link to the manifest, so that there was always a tangible html document that establishes the publication location?

Reply:
I do not think we have a choice on this one. The JSON-LD spec explicitly says that the base tag must be considered for an embedded JSON-LD...